CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ Added
 * Add ``get_entrypoint()`` method to ``ActionResourceManager`` attribute of st2client.
   #4791
 * Add support for orquesta task retry. (new feature)
+* Add config option ``scheduler.execution_scheduling_timeout_threshold_min``. Fixes #4887
```

Comment:
Changelog record should be a bit more descriptive, but not too much to keep it compact. For example:

```suggestion
* Add config option ``scheduler.execution_scheduling_timeout_threshold_min`` to better control the cleanup of scheduled actions that were orphaned. #4886
```

Reply:
This is still unresolved. I committed this change before, but your force-push re-writed it back to the previous version.